<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AvASys v1.2</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #222222;
    color: #ffffff;
    font-family: &quot;Roboto&quot;, sans-serif;
    font-size: 14px;
}

QLabel {
    color: #ffffff;
}

QPushButton {
    background-color: #333333;
    color: #ffffff;
    border-radius: 5px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #444444;
}

QLineEdit {
    background-color: #333333;
    color: #ffffff;
    border: 1px solid #444444;
    padding: 10px;
}

QTextEdit {
    background-color: #333333;
    color: #ffffff;
    border: 1px solid #444444;
    padding: 10px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidgetMain">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>600</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="pageLogin">
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <widget class="QLineEdit" name="lineEditUsername">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>35</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0" alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="pushButtonLogin">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="lineEditPassword">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>35</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="page_2Menu">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0" colspan="4">
       <widget class="QStackedWidget" name="stackedWidgetMenu">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page_3">
         <widget class="QWebEngineView" name="webEngineViewMap">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>1000</width>
            <height>530</height>
           </rect>
          </property>
          <property name="url">
           <url>
            <string>about:blank</string>
           </url>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
